pkg/rancher-desktop/components/Nav.vue

```diff
@@ -172,4 +181,19 @@ a {
   font-size: 0.75rem;
 }
 
+@media (prefers-color-scheme: dark) {
+  .extension-icon {
+    &:not(.no-theme) {
```

Comment:
Rather than calling this `.no-theme` (and making it opt-out), it should instead be opt-in (that is, only set on icons that need this treatment) and have a more obvious name (going by the comment on line 124, something like `.known-monochrome`).